tests/test_matrix.py

```diff
@@ -1,4 +1,4 @@
-from algorithms.matrix import (
+from algorithms.algorithms.matrix import (
```

Comment:
When you add this I had to go about and add an init.py in the main folder... Will it cause merge conflicts?